spec/src/main/asciidoc/UsingFacesInWebApplications.adoc

```diff
@@ -234,17 +223,18 @@ _ViewHandler.FACELETS_SUFFIX_PARAM_NAME_ for the complete specification.
 this param is set, the runtime must interpret it as a semicolon (;)
 separated list of strings that is used to forcibly declare that certain
 pages in the application must be interpreted as using Facelets,
-regardless of their extension. The runtime must also consider the
-_facelets.VIEW_MAPPINGS_ param name as an alias to this param name for
-backwards compatibility with existing facelets applications. See the
+regardless of their extension. See the
 javadocs for the symbolic constant
 _ViewHandler.FACELETS_VIEW_MAPPINGS_PARAM_NAME_ for the complete
 specification.
 
-* _jakarta.faces.FULL_STATE_SAVING_VIEW_IDS_ --
+* +++<del>+++_jakarta.faces.FULL_STATE_SAVING_VIEW_IDS_ --
 The runtime must interpret the value of this parameter as a comma
 separated list of view IDs, each of which must have their state saved
-using the state saving mechanism specified in pre-Jakarta Faces JSF 1.2 (under the JCP).
+using the state saving mechanism specified in pre-Jakarta Faces JSF 1.2 (under the JCP).+++</del>+++
+*Deprecated since 4.1:*
+Full state saving will be removed in favor of partial state saving in order to keep the spec simple.
+Therefore specifying full state saving view IDs via this context parameter will not anymore be an option.
```

Comment:
Therefore specifying full state saving view IDs via this context parameter will not be an option anymore.